doc/content/modules/user-manual/pages/release-notes/2025.2.0.adoc

```diff
@@ -103,6 +103,11 @@ image::release-notes-diagrams-export-as-png.png[Export diagram as PNG, width=50%
 
 - `Membership#isDistinguishableFrom` derived attribute has been implemented in the metamodel.
 
+- SysML textual export now correctly exports the following concepts :
+* ConcernDefinition
```

Comment:
Could you with surround SysMLv2 concepts with '`' please?

Reply:
Done.